installer.tpl.sh

```diff
@@ -9,6 +9,13 @@ fi
 
 repo_type="{{ rtype }}"
 repo_path=""
+arch=""
+gc64=""
+
+detect_arch ()
```

Comment:
There is no need in this function because there is the `arch` command. So just use `$(arch)` where it is needed.
```bash
$ arch
x86_64
```

Reply:
removed the function, but saved `uname -m` as it was before